bucket/theia-ide.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://raw.github.com/ScoopInstaller/Scoop/master/schema.json",
+    "version": "1.49.1",
```

Comment:
```suggestion
    "version": "1.49.100",
```

Reply:
In `About` page, the version is 1.49.100.

Reply:
The theia project is version 1.49.1 which is a framework, when came to IDE, the version is 1.49.100.

Reply:
ohww ohkay. got it.

Reply:
hey so, should checkver use `latest.yml` file ?